Automated Smoketest: :x:
OS: centos 7.6.1810 (Linux 3.10.0-957.1.3.el7.x86_64)
 GCC: gcc (GCC) 7.3.1 20180303 (Red Hat 7.3.1-5)
 Host: ip-10-20-0-225.ca-central-1.compute.internal
Sha: 364e30749ae2819ea6aecf12d7fd0447ae8dfa3b
Number of Error(s): 3
make[2]: *** [esp/src/CMakeFiles/eclwatch_fetchdeps.dir/build.make:61: esp/src/lib/package-lock.json] Error 1
make[1]: *** [CMakeFiles/Makefile2:10281: esp/src/CMakeFiles/eclwatch_fetchdeps.dir/all] Error 2
make: *** [Makefile:163: all] Error 2
Get ECLWatch build result
ReportTimes.
Finish at: 2021-03-23_14-50-01
time logs started
Prep time   :     8 sec (00:00:08)
Build time  :   323 sec (00:05:23)
Package time:     0 sec (00:00:00)
Install time:     0 sec (00:00:00)
Start time  :     0 sec (00:00:00)
Test time   :     0 sec (00:00:00)
Stop time   :     0 sec (00:00:00)
Summary     :   331 sec (00:05:31)
time logs end

[Build error:](https://gist.github.com/31f734b511831cf6d5cc276a10a4af05)
Build log [Build log](https://gist.github.com/HPCCSmoketest/31f734b511831cf6d5cc276a10a4af05#file-result-21-03-23-14-44-18-log)

--- a/esp/src/src/Timings.ts
+++ b/esp/src/src/Timings.ts
@@ -405,6 +405,8 @@ export class WUTimelineEx extends WUTimeline {
 
     constructor() {
         super();
+        this._gantt.bucketHeight(16);
+        this.strokeWidth(0);
     }
 
     data(): any;

